docs/cmdline/run-env.md

```diff
@@ -0,0 +1,31 @@
+---
+title: terramate run-env - Command
```

Comment:
shouldn't this be different for experimental commands, to reflect the subcommand tree?

Reply:
fair enough yep